src/Compression/CompressedReadBufferBase.cpp

```diff
@@ -1,4 +1,5 @@
 #include "CompressedReadBufferBase.h"
+#include "Utilities.h"
```

Comment:
Utilities is a bad name for a header file.
It means, "I want to put random unclassified garbage there".

Reply:
I get where you're coming from, I'll come up with a proper name.